doctrine/doctrine-bundle/1.12/config/packages/dev/doctrine.yaml

```diff
@@ -0,0 +1,3 @@
+doctrine:
+    orm:
+        auto_generate_proxy_classes: true
```

Comment:
I think we need to revert back to the `prod` config that sets this to false. Because with this config, the in the `test` environment, the proxy classes would not be automatically generated

Reply:
Good catch! I've reverted to the previous behaviour, which still makes the prod config significantly shorter!